armi/reactor/blocks.py

```diff
@@ -1785,6 +1785,13 @@ def coords(self):
             round(y, units.FLOAT_DIMENSION_DECIMALS),
         )
 
+    def cornersUp(self):
+        """Determine if the hex shape of is corners up or flats up, in relation to the Y axis."""
+        if self.spatialGrid is None:
+            return None
```

Comment:
I guess this is false-like? But returning `None` from a bool-like method feels weird. 

Would it make more sense to return False or error here?

Reply:
A Python `None` is a null value. It is just as valid to use `None` for a null value when a return type is an `int`, `numpy.ndarray` or `bool`.

Seems standard enough to return an empty value from a method.

What would you rather do?  I don't want to return an error, because this does not seem like a scenario in which I want to kill a simulation.

Reply:
hmmm